### PR TITLE
kopia: init at 0.5.2

### PR DESCRIPTION
**Pull request by bbigras**
###### Things done

I only tested a "filesystem" backup for now.

Thanks to @Infinisil for the help with the "vendor" thing.

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/backup/kopia/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  };
+
+  vendorSha256 = "11az7zgwzbcx4dknwqiwmdbrbkdzhpwzqnyk8vw9mkbda0xaif3k";
+  subPackages = [ "." ];
+
+  postConfigure = ''
+    # make 'vendor' writable
+    cp -L -r vendor tmp-vendor
+    rm -rf vendor
+    mv tmp-vendor vendor
```
Comment (dasj19) [inline pkgs/tools/backup/kopia/default.nix:21]: why not just `chmod +w vendor`?

**Comment by dasj19** [comment]
Built with `nix-shell -p nixpkgs-review --run "nixpkgs-review pr 94462"` and runs fine. You can thick the 3rd checkbox in the description.

File: pkgs/tools/backup/kopia/default.nix (hunk 1 above)
Comment (bbigras) [reply pkgs/tools/backup/kopia/default.nix:21]: It's a symlink pointing to the nix store. I should make the comment clearer.

**Review by jonringer — APPROVED** [review]
LGTM
shows usage

```
https://github.com/NixOS/nixpkgs/pull/94462
1 package built:
kopia
```

**Comment by bbigras** [comment]
Thank you very much.

**Pull request merged**